tests/functional/monitoring/prometheus/alerts/test_noobaa_bucket_error_state_mode.py

```
@@ -0,0 +1,557 @@
+        bucket = MCGCLIBucket(bucket_name, mcg=mcg_obj, quota="2Gi")
+        log.info(f"Created bucket {bucket_name} with 2Gi quota")
+
+        try:
+            _upload_data_to_bucket(awscli_pod, bucket_name, mcg_obj, 5, 500)
+            log.info(
+                f"Uploaded ~2.5GB to bucket {bucket_name} "
+                f"(quota 2Gi) to trigger EXCEEDING_QUOTA"
+            )
+
+            run_time = 60 * 7
+            log.info(f"Waiting {run_time}s for bucket mode to propagate")
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)